system/Email/Email.php

```diff
@@ -760,43 +763,43 @@ protected function stringToArray($email)
     /**
      * @param string $str
      *
-     * @return Email
+     * @return $this
      */
     public function setAltMessage($str)
     {
-        $this->altMessage = (string) $str;
```

Comment:
Why are we getting rid of casting here?

Reply:
Because it is already `string` in the PHPDoc.

Reply:
PHPDoc doesn't enforce types at runtime. Since we're not using a strict type declaration here, it's safer to leave the explicit cast to string.